WordPress/Classes/ViewRelated/Pages/PageListCell.swift

```diff
@@ -113,6 +113,7 @@ final class PageListCell: UITableViewCell, AbstractPostListCell, PostSearchResul
         setupLabels()
         setupFeaturedImageView()
         setupEllipsisButton()
+        setupIcon()
```

Comment:
This code was reported as unused, but it's a mistake. The issue was harmless: this icon was a tiny bit larger than expected:

<img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/1567433/8ce82a4b-b064-4515-a021-d190976b70ca" width="320px">